packaging.mk

```diff
@@ -29,21 +29,31 @@ DOCKER_IMAGES := \
 	build/docker/apm-server-$(APM_SERVER_VERSION).txt \
 	build/docker/apm-server-$(APM_SERVER_VERSION)-SNAPSHOT.txt \
 	build/docker/apm-server-ubi-$(APM_SERVER_VERSION).txt \
-	build/docker/apm-server-ubi-$(APM_SERVER_VERSION)-SNAPSHOT.txt
+	build/docker/apm-server-ubi-$(APM_SERVER_VERSION)-SNAPSHOT.txt \
+	build/docker/apm-server-wolfi-$(APM_SERVER_VERSION).txt \
+	build/docker/apm-server-wolfi-$(APM_SERVER_VERSION)-SNAPSHOT.txt
 
 build/docker/%.txt: DOCKER_IMAGE_TAG := docker.elastic.co/apm/apm-server:%
 build/docker/%.txt: VERSION := $(APM_SERVER_VERSION)
 build/docker/%-SNAPSHOT.txt: VERSION := $(APM_SERVER_VERSION)-SNAPSHOT
 build/docker/apm-server-ubi-%.txt: DOCKER_BUILD_ARGS+=--build-arg BASE_IMAGE=docker.elastic.co/ubi9/ubi-minimal
+build/docker/apm-server-wolfi-%.txt: DOCKER_BUILD_ARGS+=--build-arg BASE_IMAGE=docker.elastic.co/wolfi/chainguard-base:20230214 --build-arg BASE_GOLANG_IMAGE=docker.elastic.co/wolfi/go
```

Comment:
I need to figure out how to add a conditional so this will only run if `WOLFI=true` then I can configure the ci workflows accordingly.